### PR TITLE
fix: validate widget with schemaProperties and emit updated widget info

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
Please review in order by commits

### CODE REVIEW

File: apps/web/src/services/dashboards/shared/dashboard-widget-input-form/DashboardWidgetInputForm.vue [hunk 1]
```diff
@@ -182,39 +180,20 @@ const initSchemaAndFormData = (widgetConfigId: string, widgetKey?: string) => {
...
-
-/* sync to widget form store */
-watch(() => state.schemaFormData, (schemaFormData, before) => {
-    if (isEmpty(state.widgetOptionsJsonSchema)) return;
-    if (state.widgetOptionsJsonSchema.properties?.cost_group_by && !schemaFormData?.cost_group_by) return;
-    if (state.widgetOptionsJsonSchema.properties?.asset_group_by && !before?.asset_group_by) return;
-    widgetFormStore.updateInheritOptionsAndWidgetOptionsByFormData(schemaFormData);
-}, { immediate: true });
-watch(() => state.isAllValid, (_isAllValid) => {
-    widgetFormStore.$patch({ isValid: _isAllValid });
```
Comment (yuda110) [inline apps/web/src/services/dashboards/shared/dashboard-widget-input-form/DashboardWidgetInputForm.vue:217]: praise: ho!!!!!!🙌

File: apps/web/src/services/dashboards/widgets/_hooks/use-widget/use-widget-state.ts [hunk 2]
```diff
@@ -53,6 +53,8 @@ export function useWidgetState(props: WidgetProps) {
         dashboardSettings: toRef(props, 'dashboardSettings'),
         dashboardVariablesSchema: toRef(props, 'dashboardVariablesSchema'),
         dashboardVariables: toRef(props, 'dashboardVariables'),
+        title: toRef(props, 'title'),
+        schemaProperties: toRef(props, 'schemaProperties'),
```
Comment (piggggggggy) [inline apps/web/src/services/dashboards/widgets/_hooks/use-widget/use-widget-state.ts:57]: 😄

**Review by piggggggggy — APPROVED** [review]
LGTM!

**Review by yuda110 — APPROVED** [review]
LGTM!!!

**Pull request merged**